app/services/notification_subscriptions/unsubscribe.rb

```diff
@@ -17,24 +17,31 @@ def call
 
     def unsubscribe_subscription
       return { errors: "Subscription ID is missing" } if subscription_id.nil?
```

Comment:
I found this a little odd that we handle cases where the subscription is nil and where the subscription id is nil. Hence, just wanted to confirm, can we have a case where the subscription is present but the subscription id is nil?

Reply:
These guard clauses have been in here [since nearly the beginning](https://github.com/forem/forem/blob/f010b952fd7f7293b5e9ff5b04cdd0e9c2627c3c/app/services/notification_subscriptions/unsubscribe.rb) but we don't need both of them now -- good catch!